install/kubernetes/helm/istio/charts/pilot/templates/deployment.yaml

```diff
@@ -104,6 +104,11 @@ spec:
               fieldRef:
                 apiVersion: v1
                 fieldPath: metadata.namespace
+          - name: INSTANCE_IP
```

Comment:
I assume this and the others are for the case where mixer may be deployed in multiple regions ?

We need to put a comment in some places...

Reply:
@costinm I am not sure why this is needed.  @sbezverk indicated proxy-agent is broken without it, however, Istio master has worked well for me for months.

While the use case of deploying mixer in several regions may be a thing post 1.0, its not  a priority for 1.0 so if this change is made lets make it for correct reasons :)

Cheers
-steve

Reply:
This is needed for zone aware routing when its eventually fully enabled. It wont break anything currently other than logging some unhelpful errors.